slasher/db/db.go

```diff
@@ -76,8 +87,23 @@ func NewKVStore(dirPath string) (*Store, error) {
 		}
 		return nil, err
 	}
-
-	kv := &Store{db: boltDB, databasePath: dirPath}
+	if cacheItems == 0 {
+		cacheItems = 20000
+	}
+	if maxCacheSize == 0 {
+		maxCacheSize = 2 << 30 //(2GB)
+	}
+	spanCache, err := ristretto.NewCache(&ristretto.Config{
+		NumCounters: cacheItems,   // number of keys to track frequency of (10M).
+		MaxCost:     maxCacheSize, // maximum cost of cache.
+		BufferItems: 64,           // number of keys per Get buffer.
+		OnEvict:     saveToDB,
+	})
+	if err != nil {
+		errors.Wrap(err, "failed to start span cache")
+		panic(err)
```

Comment:
Not sure about this panic, if you look at how we initialize beacon node db. Panic is not used there

Reply:
changed to return error

Reply:
thanks